OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -111,9 +111,9 @@ private void Pane_DblClickHandler(object sender, EventArgs e)
                 }
                 expandedImageForm.Close();
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                MessageBox.Show("Error Opening Image", "Error", MessageBoxButtons.OK);
+                MessageBox.Show(ex + "\n" + ex.Message);
```

Comment:
![error](https://user-images.githubusercontent.com/26779711/36924887-6099c89c-1e3e-11e8-84b4-656db0dd9404.PNG)

Reply:
I don't think the user needs all this information. We possibly need to give more info than previously done but this is too much.

Reply:
Oh, oops. I forgot I still had that in there. That was just so I could debug, the catch all try block was preventing me from debugging before I changed it.